libindy/src/api/wallet.rs

```diff
@@ -473,13 +477,14 @@ pub extern fn indy_close_wallet(command_handle: i32,
 /// }
 /// credentials: Wallet credentials json
 /// {
-///   "key": string, Passphrase used to derive wallet master key
+///   "key": string, Auth key for the wallet
 ///   "storage_credentials": optional<object> Credentials for wallet storage. Storage type defines set of supported keys.
 ///                          Can be optional if storage supports default configuration.
 ///                          For 'default' storage type should be empty.
-///   "key_derivation_method": optional<string> algorithm to use for master key derivation:
-///                            ARAGON2I_MOD (used by default)
-///                            ARAGON2I_INT - less secured but faster
+///   "key_derivation_method": optional<string> Type of wallet auth key:
+///                             ARAGON2I_MOD - derive secured wallet master key (used by default)
+///                             ARAGON2I_INT - derive secured wallet master key (less secured but faster)
+///                             RAW - raw wallet key master provided (skip derivation)
```

Comment:
Update description:

raw wallet key master provided (skip derivation). RAW keys can be generated with indy_generate_wallet_key call